client/pages/Dashboard.js

```diff
@@ -59,10 +63,26 @@ class Dashboard extends Component {
   }
 
 
-  addNotification(msgTitle, msg, participantId = 0) {
+  @autobind
+  loadEventsNotifications() {
+    const { events, curUser } = this.state;
+    events.forEach((event) => {
+      event.participants.forEach(
+        (participant) => {
```

Comment:
Please move this onto the line above. We want to avoid indentation of code if possible.